### DISCUSSION TITLE
Usage of  Updating multiple stores

### DISCUSSION THREAD

**Discussion by MilesLeeeeee** [pmndrs/zustand#2067] [Q&A] [answered]
## Summary
Hello，I tried to use the Slices pattern following the documentation

![image](https://github.com/pmndrs/zustand/assets/129132865/f78ab335-530f-41ac-bfe4-b453d012359f)
Updating multiple stores seems inconsistent with actual usage

i found a way from [#2045](https://github.com/pmndrs/zustand/discussions/2045)

It seems that using get() is correct?

## Link to reproduction
[https://codesandbox.io/s/recursing-proskuriakova-qmyz2n?file=/store/zustand.ts](https://codesandbox.io/s/recursing-proskuriakova-qmyz2n?file=/store/zustand.ts)

**Comment by dbritto-dev** [comment, marked as answer]
@MilesLeeeeee the usage of create slice functions inside of actions is wrong. You should always use `get`.

> Note: since you want to use get state in your create slice function you should pass `set` and `get` not only `set`. For instance: instead of `createFishSlice(set)` should be `createFishSlice(set, get)` or `createFishSlice(...a)`

**Comment by dbritto-dev** [comment]
@dai-shi seems that zustand docs page is outdated, would you mind deploying the latest version?

**Reply by dai-shi** [reply]
@CodyJasonBennett Is something wrong with the deployment?

**Reply by CodyJasonBennett** [reply]
Looks like it got stuck on some newer link syntax. Should be fixed now.

**Comment by MilesLeeeeee** [comment]
@dbritto-dev yes，Using get() is right, thanks a lot !!!
